python/lsst/ts/Dome/dome_csc.py

```diff
@@ -63,9 +67,15 @@ def __init__(
             simulation_mode=simulation_mode,
         )
 
+        stream_handler = logging.StreamHandler()
+        self.log.addHandler(stream_handler)
```

Comment:
It is very unusual for a CSC to stream log messages. Is this temporary? If it's permanent then please consider RotatingFileHandler or SysLogHandler.

Reply:
It is not temporary and this actually was recommended by Tiago. I do agree on using a RotatingFileHandler so I've changed to that.